node-notes-cli/notes.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs');
+const jsonDerulo = require('./data.json');
+
+const command = process.argv[2];
+const action = process.argv[3];
+const updateText = process.argv[4];
+
+if (command === 'read') {
+  fs.readFile('data.json', 'utf8', (err, data) => {
+    if (err) throw err;
+    console.log(jsonDerulo.notes);
+  });
```

Comment:
You can remove this readFile since you already required the data. Also, take a close look at the expected terminal output for read. Make yours match that.

Reply:
<img width="944" alt="image" src="https://user-images.githubusercontent.com/99862668/176800195-9344ae67-f3c3-4293-80e5-948b4a51d1b0.png">